app/src/main/java/com/sopt/clody/presentation/utils/extension/TimeZoneExt.kt

```
@@ -0,0 +1,60 @@
+    val hour24 = userDateTime.hour
+    val minute = userDateTime.minute
+
+    val timePeriod = if (hour24 < 12) TimePeriod.AM else TimePeriod.PM
+    val hour12 = when {
+        hour24 == 0 -> 12
+        hour24 > 12 -> hour24 - 12
+        else -> hour24
+    }
+
+    val hourFormatted = hour12.toString()
+    val minuteFormatted = String.format("%02d", minute)
```

detekt (1.23.8)

[warning] String.format("%02d", minute) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)